### PR TITLE
Translate mythbusters.

### PR DESCRIPTION
**Pull request by SD57**
Maybe "Witnesses of Dead C++"?
The rest feels OK.

### CODE REVIEW

**Comment by SD57** [comment]
@salmer @DmitrievDmitriyA review pls.

**Comment by salmer** [comment]
> Maybe "Witnesses of Dead C++"? The rest feels OK.

I'd say the original name describes an article's intention better. The article contains a collection of biases about C++ not only related to the "death" of the language.

File: English/Mythbusters.md [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
+C++ made its way to the top of a wide range of ratings of programming languages, and it is scoring, moreover, popularity points, for example in the [Tiobe](https://www.tiobe.com/tiobe-index/) index. C++ got its notorious "dead language" badge during the noughties, while it was dormant and the language standardization committee fell off the radar. But the language is experiencing a renaissance since the C++11 standard. It is arduously getting new features and functionality, every three years. Many problems claimed by the "dead C++ witnesses" have been solved, but since such specialists have stopped developing using C++ - or got a smattering of C++ during courses (from those very "witnesses") - they continue repeating and reiterating the myths and legends of the horrors of C++. 
 
-## "Настоящие программисты" начинают изучать C++ сразу под Linux/Vim/gcc
+## Reals programmers learn C++ using Linux/Vim/gcc
```
Comment (salmer) [inline English/Mythbusters.md:9]: Real programmers? Typo issue in Real"s"?

**Comment by SD57** [comment]
> > Maybe "Witnesses of Dead C++"? The rest feels OK.
> 
> I'd say the original name describes an article's intention better. The article contains a collection of biases about C++ not only related to the "death" of the language.

I meant the first paragraph, not the title.

File: English/Mythbusters.md [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
 
-Это байка также идет от "старогвардейцев", которые перестали писать на языке до появления стандарта C++11, либо от тех, кто поверхностно его изучал в ВУЗе без погружения в последние стандарты. Современный C++ имеет в составе своей библиотеки набор примитивов, который отвечает за  автоматическое выделение и освобождение памяти. Контроль за выделением памяти отошел на второй план. Во многих компаниях и командах вы также встретите правило: "не используйте сырых указателей". Опять же, не пренебрегайте современным инструментарием и санитайзерами. Они способны детектировать потенциальную утечку памяти ещё на этапе исходного кода.
+This is another urban legend from the Old Guard that had stopped writing C++ before C++11 or those who superficially learned it in university disregarding the latest standards. Modern C++ contains a set of primitives in its standard library which are responsible for the automatic memory allocation and deallocation. The manual memory management fell by the wayside. Many teams and companies even have the rule: "No raw pointers". Once again, do not neglect the modern tools and sanitizers: they can detect possible memory leaks at the source code level.
```
Comment (salmer) [inline English/Mythbusters.md:41]: Old Guard in quotes as it's done before?

File: English/Mythbusters.md (hunk 1 above)
Comment (salmer) [inline English/Mythbusters.md:9]: Real programmers? Typo issue in Real"s"?

**Review by salmer — CHANGES_REQUESTED** [review]

**Comment by salmer** [comment]
2 comments are left only, plz check. In general, the article looks perfect!

**Comment by SD57** [comment]
> 2 comments are left only, plz check. In general, the article looks perfect!

I agree with the changes. Could you please apply them? I'm on the phone right now - not sure when  I'll get to the laptop.

**Pull request merged**